### PR TITLE
Improvement: Use new api

### LINKED ISSUE #84: Feature: update backend server
- Update all services with call to new api
- Add display of current user's picture in UserButton
- Add display of all users's picture in UsersTable

### PR DESCRIPTION
**Pull request by justinehell**
Resolves #84 
Use new api

### CODE REVIEW

File: global.config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "backendUrl": "http://localhost:1337/",
+  "backendUrl": "https://localhost:8443",
```
Comment (Zorin95670) [inline global.config.json:2]: May be you need to update README too

File: src/boot/axios.js [hunk 2]
```diff
@@ -1,36 +1,21 @@
...
+const api = axios.create({ baseURL: '/api' });
 
 /**
  * Redirect on Leto-modelizer on status 503, otherwise throw error.
  * @param {Error} error - Error from axios.
  */
 function manageError(error) {
   if (error.response.status !== 503) {
-    throw new Error(error.response.data.error);
+    throw error;
```
Comment (Zorin95670) [inline src/boot/axios.js:18]: keep error.response.data

File: src/router/index.js [hunk 3]
```diff
@@ -39,19 +41,10 @@ export default route(async () => {
...
-      localStorage.setItem('sessionToken', matches[1]);
-      // Redirect to admin without token in url.
-      window.location.href = window.location.href.replaceAll(/\?.+$/g, '');
-    } else if (getUserSessionToken()) {
-      initUser(getUserSessionToken());
-      next();
-    } else {
-      // Redirect to leto-modelizer in case of not connected user.
-      window.location.href = `${process.env.LETO_MODELIZER_URL}/admin`;
+    if (!userStore.login) {
```
Comment (Zorin95670) [inline src/router/index.js:44]: use isEmpty

File: src/composables/UserAuthentication.js [hunk 4]
```diff
@@ -1,55 +1,27 @@
...
 
-  userStore.roles = roles.map(({ name }) => name);
+  userStore.name = user.name;
+  userStore.login = user.login;
+  userStore.email = user.email;
+  userStore.permissions = await UserService.getMyPermissions();
   userStore.ready = true;
 
-  if (!userStore.roles.includes('admin')) {
+  if (!userStore.permissions.some(({ action, entity }) => action === 'ACCESS' && entity === 'ADMIN')) {
```
Comment (Zorin95670) [inline src/composables/UserAuthentication.js:24]: missing test

File: global.config.json (hunk 1 above)
Comment (justinehell) [reply global.config.json:2]: done

File: src/boot/axios.js (hunk 2 above)
Comment (justinehell) [reply src/boot/axios.js]: done

File: src/router/index.js (hunk 3 above)
Comment (justinehell) [reply src/router/index.js]: update

File: src/composables/UserAuthentication.js (hunk 4 above)
Comment (justinehell) [reply src/composables/UserAuthentication.js:24]: done

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**